CHANGELOG.rst

```diff
@@ -1,6 +1,14 @@
 Changelog
 =========
 
+
+Work in progress
```

Comment:
```suggestion
master
```

Another option which makes clear that the work is done, but it's setting unreleased in master...